index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+    <h1>Hello Microverse ! </h1>
```

Comment:
- [ ]  As stated the project requirement Kindly make your added A .css file that includes `one class` with styles for your header (make it in your favorite color).

![Capture](https://user-images.githubusercontent.com/93598189/205635210-2d9d6383-4a6e-4e77-84f3-f531ccaea143.PNG)